src/docs/reference/crash-reporting.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Flutter crash reporting
+---
+
+If you have not opted-out of Flutter's analytics and crash reporting, when a
+`flutter` command crashes it attempts to send a crash report to Google in order
+to help Google contribute improvements to Flutter over time. A crash report
+may contain the following information:
+
+* The name and version of your local operating system
+* The version of Flutter
+* The runtime type of the error, e.g. `StateError` or `NoSuchMethodError`
```

Comment:
```suggestion
* The runtime type of the error, for example `StateError` or `NoSuchMethodError`.
```

Reply:
https://developers.google.com/style/word-list#letter-i 
(see entry for avoiding using e.g.)